box_packer.rb

```diff
@@ -102,8 +102,8 @@ def break_up_space(space, placement)
         {
           dimensions: [
             placement[:dimensions][0],
-            space[:dimensions][1],
-            space[:dimensions][2] - placement[:dimensions][1]
+            space[:dimensions][1] - placement[:dimensions][1],
+            placement[:dimensions][2]
```

Comment:
Fix for overhangs. This ensures that if a box is stacked on top of another, the top box's width and length can't be greater than the lower box's.